yew-router/src/router.rs

```diff
@@ -185,8 +185,10 @@ where
         }
     }
 
-    fn rendered(&mut self, _first_render: bool) {
-        self.router_agent.send(RouteRequest::GetCurrentRoute);
+    fn rendered(&mut self, first_render: bool) {
+        if first_render {
```

Comment:
I think a better fix would be to check if `Msg::UpdateRoute` contains the same route that's currently shown. @hgzimmerman thoughts?

Reply:
Actually we should move `self.router_agent.send(RouteRequest::GetCurrentRoute);` to the `create(..)` method

Reply:
Yeah, I think it should belong in `create` instead.

Reply:
ok I can try to move it